core/src/test/scala/unit/kafka/server/ApiVersionsRequestTest.scala

```diff
@@ -94,7 +94,7 @@ class ApiVersionsRequestTest(cluster: ClusterInstance) extends AbstractApiVersio
   }
 
   @ClusterTemplate("testApiVersionsRequestIncludesUnreleasedApisTemplate")
-  @ClusterTest(types = Array(Type.KRAFT, Type.CO_KRAFT), serverProperties = Array(
+  @ClusterTest(types = Array(Type.KRAFT, Type.CO_KRAFT), metadataVersion = MetadataVersion.IBP_3_9_IV0, serverProperties = Array(
```

Comment:
`ClusterTest` in trunk has `IBP_3_9_IV0` as default value instead of `IBP_3_8_IV0`. This took me a while to detect as I didn't understand why this test was failing.

Reply:
Actually, ClusterTest.metadataVersion() should default to latest testing MV, which is IBP_3_9_IV0. Could we change that and remove setting metadataVersion in this test and the one above?